src/tools/kata-ctl/Makefile

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include ../../../utils.mk
+
+.DEFAULT_GOAL := default
+default: build
+
+build: logging-crate-tests
+	@RUSTFLAGS="$(EXTRA_RUSTFLAGS) --deny warnings" cargo build --target $(TRIPLE) --$(BUILD_TYPE)
+
+logging-crate-tests:
+	make -C $(CWD)/../../libs/logging
```

Comment:
Nit: This doesn'twork because the `logging` crate doesn't have a `Makefile` (but `cargo build` still works ;)

For now, I'd be tempted to simply remove all the `logging-crate-tests` details from this file.